runtime/nls/shrc/j9shr.nls

```diff
@@ -6981,12 +6981,12 @@ J9NLS_SHRC_CC_EXTRA_STARTUPHINTS_SET.system_action=The JVM continues.
 J9NLS_SHRC_CC_EXTRA_STARTUPHINTS_SET.user_response=No action required. This message is for information only.
 # END NON-TRANSLATABLE
 
-J9NLS_SHRC_CM_PRINTSTATS_STARTUP_HINTS=additional startup hints allowed    %*.c= %u
-# START NON-TRANSLATABLE
-J9NLS_SHRC_CM_PRINTSTATS_STARTUP_HINTS.sample_input_1=0
-J9NLS_SHRC_CM_PRINTSTATS_STARTUP_HINTS.sample_input_2=
-J9NLS_SHRC_CM_PRINTSTATS_STARTUP_HINTS.sample_input_3=8
-J9NLS_SHRC_CM_PRINTSTATS_STARTUP_HINTS.explanation=NOTAG
-J9NLS_SHRC_CM_PRINTSTATS_STARTUP_HINTS.system_action=
-J9NLS_SHRC_CM_PRINTSTATS_STARTUP_HINTS.user_response=
+J9NLS_SHRC_CM_PRINTSTATS_NUM_EXTRA_STARTUP_HINTS=# Additional startup hints allowed  %*.c= %u
```

Comment:
I changed J9NLS_SHRC_CM_PRINTSTATS_STARTUP_HINTS (which is added a few days ago) assuming it is not translated yet.

Reply:
Not sure it would matter if it was, I could just run the translation again.